types/src/block/block.rs

```diff
@@ -307,7 +285,7 @@ pub mod dto {
     }
 
     impl Block {
-        fn _try_from_dto(value: &BlockDto) -> Result<BlockBuilder<u64>, DtoError> {
+        fn _try_from_dto(value: &BlockDto) -> Result<BlockBuilder, DtoError> {
```

Comment:
is the _ prefix really needed here?

Reply:
Yeah it's just a helper function to factorise the 2 other ones

Reply:
Was already there anyway, bit outside of the context of this PR